libnetwork/controller.go

```diff
@@ -550,6 +550,9 @@ func (c *Controller) NewNetwork(networkType, name string, id string, options ...
 	if (caps.DataScope == scope.Global || nw.scope == scope.Swarm) &&
 		c.isSwarmNode() && !nw.dynamic {
 		if c.isManager() {
+			if !nw.enableIPv4 {
+				return nil, types.ForbiddenErrorf("IPv4 cannot be disabled in a Swarm scoped network")
```

Comment:
Hm.. on that matter; I see we're already using `types.ForbiddenErrorf` in various places, but wondering if we need to review those; it looks like those match `errdefs.ErrForbidden`, which will produce a `403` on the API. 

That's still a 4xx error, and I don't think we have code  that would mistake it for an authentication issue, but wondering if these should be a `errdefs.ErrInvalidParameter` 🤔

Reply:
Yes, I stuck with the error type that was used nearby for similar bad-config problems.

Reply:
It's now an `errdefs.ErrInvalidParameter`.